src/components/LeftPanel/LeftPanel.js

```diff
@@ -35,13 +38,30 @@ class LeftPanel extends React.Component {
     return panel === this.props.activePanel ? 'flex' : 'none';
   };
 
+  onWidthChange = width => {
+    if (isIE11 || isIEEdge) {
+      this.props.updateLeftPanel(width);
```

Comment:
Do you think naming the function as `updateLeftPanelWidth` or `setLeftPanelWidth` would be better options?

Reply:
Sorry.. And do you think we need some comments here? Explain why in IE and Edge we are updating the width in redux while in other browsers we don't

Reply:
Your right it should be 'setLeftPanelWidth'. Also I will add a comment explaining why we are doing this for IE and Edge